# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rarpd/install.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- rarpd/Makefile	2000-06-03 21:43:45.000000000 +0200
+++ rarpd-fixed/Makefile	2006-06-28 22:38:30.000000000 +0200
@@ -12,4 +12,5 @@
 	rm -f $(OBJ) rarpd
 
 install:
-	install -s -u root -m 0775 rarpd ${PREFIX}/sbin
+	install -m 0775 rarpd ${PREFIX}/sbin
+	install rarpd.8 $(PREFIX)/man/man8/
